frontends/beta-frontend/pages/team/index.tsx

```diff
@@ -1,39 +1,50 @@
 import type { NextPage } from 'next'
-import Head from 'next/head'
 import { Box } from '@mui/material'
 import Grid from '@/shared/components/Grid/Grid'
 import { SampleTeam } from '@/data/SampleTeam'
 import TeamDetails from '@/shared/components/TeamDetails'
 import PlayerTactics from '@/shared/components/PlayerTactics'
 import TeamTactics from '@/shared/components/TeamTactics'
-import { samplePlayerColumn } from '@/data/samplePlayerColumn'
-// import { getPlayers } from '../api/players/players'
-import useSWR from "swr";
-import { connectorAPI } from "@/libs/fetcher";
+import { teamColumn } from '@/shared/components/Grid/TeamColumn'
+import useSWR from "swr"
 
-const Team: NextPage = () => {
-  const { data, error, isLoading } = useSWR('/player?size=40&page=0', connectorAPI)
-  if (error) return <div>failed to load</div>
-  if (isLoading) return <div>loading...</div>
+const fetcher = (url: string) => fetch(url).then((res) => res.json());
+const API = "http://localhost:8082/player?size=40&page=0";
+
+export async function getServerSideProps() {
+  const repoInfo = await fetcher(API);
+  return {
+    props: {
+      fallback: {
+        [API]: repoInfo
+      }
+    }
+  };
+}
+
+interface TeamProps {
+  fallback: any;
+}
+
+const Team: NextPage<TeamProps> = ({fallback}) => {
+  const { data, error } = useSWR(API);
+  console.log(fallback[API])
 
-  // console.log(typeof data)
-  // console.log(data)
-  // data.forEach((element: any) => console.log(element))
+  console.log("Is data ready?", !!data);
+  console.log(data)
+
+  if (error) return <div>failed to load</div>
 
   return (
     <>
-      <Head>
-        <title>Kjeldsen</title>
-        <meta name="description" content="Generated by create next app" />
-      </Head>
       <Box>
         <Box sx={{ display: 'flex', marginBottom: '2rem', alignItems: 'center' }}>
           <TeamDetails {...SampleTeam} />
           <PlayerTactics />
           <TeamTactics />
         </Box>
         <Box sx={{ minWidth: '80vw' }}>
-          <Grid rows={data} columns={samplePlayerColumn} />
+          <Grid rows={fallback[API]} columns={teamColumn} />
```

Comment:
As I correctly understand SWR API you should use `data` here, not `fallback[API]` if you want to use cached value. 
Does not it work?